requirements.txt

```diff
@@ -1,3 +1,5 @@
-gunicorn
-eventlet==0.30.2
-greenlet>=0.4.14
\ No newline at end of file
+https://github.com/benoitc/gunicorn/archive/refs/heads/master.zip#egg=gunicorn==20.1.0
+https://github.com/eventlet/eventlet/archive/master.zip
+greenlet>=0.4.14
+django-utils-six==2.0
```

Comment:
Is this actually needed anywhere?

Reply:
django-utils-six==2.0 is still needed currently